src/test/ui/proc-macro/proc-macro-abi.rs

```diff
@@ -0,0 +1,28 @@
+#![crate_type = "proc-macro"]
+#![allow(warnings)]
+
+extern crate proc_macro;
+use proc_macro::TokenStream;
+
+#[proc_macro]
+pub extern "C" fn abi(a: TokenStream) -> TokenStream {
+    //~^ ERROR proc macro functions may not be `extern
```

Comment:
Why not just put the whole error here?

Reply:
I'm not sure; I remember having difficulty escaping the quotes, but it works now.